src/Raven.Server/Rachis/Follower.cs

```diff
@@ -16,8 +19,10 @@
 using Sparrow.Utils;
 using Voron;
 using Voron.Data;
+using Voron.Data.BTrees;
 using Voron.Data.Tables;
 using Voron.Impl;
+using StreamReader = Raven.Server.Rachis.Remote.StreamReader;
```

Comment:
This is likely to be confusing WRT the `System.IO.StreamReader`, better to give a more specific name